clients/fides-js/src/components/DataUseToggle.tsx

```diff
@@ -0,0 +1,78 @@
+import { ComponentChildren, VNode, h } from "preact";
+import { useDisclosure } from "../lib/hooks";
+import Toggle from "./Toggle";
+
+interface DataUse {
+  key: string;
+  name?: string;
+}
+
+const DataUseToggle = ({
```

Comment:
changed `NoticeToggle` to `DataUseToggle` to make it less specific to notices. It takes an obj with a `key` and `name` instead of a notice object so that the TCF models can in theory use it as well